guides/doc-Monitoring_Guide/topics/proc_identifying-available-metrics.adoc

```diff
@@ -0,0 +1,32 @@
+[id=identifying-available-metrics_{context}]
+= Identifying Available Metrics
+
+* To list all metrics available via PCP, enter the following command:
++
+----
+# pminfo
+----
+
+* To list all {Project} metrics and their descriptions, enter the following command:
++
+----
+# foreman-rake telemetry:metrics
+----
+
+* To list the archived metrics, enter the following command:
++
+----
+# less /var/log/pcp/pmlogger/$(hostname)/pmlogger.log
+----
+
+* The pmlogger daemon archives data as it is received, according to its configuration. To confirm the active archive file, enter the following command:
++
+----
+# pcp | grep logger
+----
++
+The output includes the file name of the active archive file, for example:
++
+----
+/var/log/pcp/pmlogger/satellite.example.com/20180814.00.10
```

Comment:
does the upstremize script need to be updated to catch these, or is it best not to do that because when it is a directory, it may very well be either /foreman or /satellite without being updated downstream?

Reply:
I think we should handle these manually, this is special case I'd say where URL is in a path.